src/test/java/com/kodilla/ecommercee/domain/ProductTest.java

```diff
@@ -0,0 +1,171 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.CartRepository;
+import com.kodilla.ecommercee.repository.ProductGroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.transaction.Transactional;
+import java.math.BigDecimal;
+import java.util.Optional;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ProductTest {
+    @Autowired
+    ProductRepository productRepository;
+    @Autowired
+    CartRepository cartRepository;
+    @Autowired
+    ProductGroupRepository productGroupRepository;
+
+    @Test
+    public void saveProduct() {
+        //Given
+        Product product = new Product("shoes", "black and winter", new BigDecimal(100), true);
+        //When
+        productRepository.save(product);
+        //Then
+        Long productId = product.getId();
+        Optional<Product> saveProduct = productRepository.findById(productId);
+        Assert.assertTrue(saveProduct.isPresent());
```

Comment:
proponuję używać w takich przypadkach biblioteki assertJ i można to zapisać wowczas:
`assertThat(savedProduct).isPresent();`